guide/14-deep-learning/model_extension_guide.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>We need a loss function to calculate how much is the model <strong>prediction</strong> deviating from the ground truth.&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/987/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>